index.html

```diff
@@ -7,14 +7,181 @@
+                            <p>lorem Lorem ipsum dolor sit, amet consectetur adipisicing elit. Beatae, nisi neque quod nemo ea obcaecati nihil reiciendis asperiores, consectetur facere, vitae accusamus! Sit enim excepturi eum laboriosam cupiditate, tempore quasi.</p>
+                        </div>
+                    </div>
+                    <div class="about-col-6">
+                        <div class="card">
+                            <img src="/images/divided.jpg" alt="hand">
+                        </div>
+                    </div> 
+                    <div class="about-col-6">
+                        <div class="card">
+                            <img src="/images/hand.jpg" alt="hand">
+                        </div>
+                    </div> 
+                    <div class="col-12 about-col-6">
+                        <div class="content">
+                            <h1>BUY ONLINE NOW & GET 10% OFF!</h1>
+                            <div class="button">Shop Now</div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+        </div>
+    </section>
+   <!-- FORM -->
+<section>
+    <div class="container">
+        <div class="row">
+            <div class="col-12 md">
+                <div class="request">
+                    <h4 class="form-title">Contact Form</h4>
 
-    <!-- Add your markup here -->
+                    <div class="form-container">
+                      <form action="/action_page.php">
+                        <label for="fname">First Name</label>
+                        <input type="text" id="fname" name="firstname" placeholder="Your name..">
+                    
+                        <label for="lname">Last Name</label>
+                        <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+                    
+                        <label for="country">Country</label>
+                        <select id="country" name="country">
+                          <option value="australia">Australia</option>
+                          <option value="canada">Canada</option>
+                          <option value="usa">USA</option>
+                        </select>
+                    
+                        <label for="subject">Subject</label>
+                        <textarea id="subject" name="subject" placeholder="Write something.." style="height:200px"></textarea>
+                    
+                        <input type="submit" value="Submit">
+                      </form>
+                    </div>
+                </div>
+            </div>
+        
+   <!-- MAP -->        
+            <div class="row-3">
+                <div class="col-12 md-col-6">
+                    <div class="card">
+                        <img src="/images/map.PNG" alt="hand">
+                    </div>   
+                </div> 
+                <div class="col-12 md-col-6"> 
+                <div class="row">
```

Comment:
Careful with your identation here - the row class is a child of your col class here